Lilu/Sources/kern_devinfo.cpp

```diff
@@ -256,7 +256,7 @@ void DeviceInfo::grabDevicesFromPciRoot(IORegistryEntry *pciRoot) {
 
 					pciiterator->release();
 
-					if (v.video) {
+					if (((v.audio && strcmp(v.audio->getName(), "AZAL")) || !v.audio) && v.video) { // check for some AMDZEN cases
```

Comment:
Please add != 0 near strcmp and provide a more descriptive comment why this change in this needed right in the code prior to the if condition.

Like "On AMD Zen platforms… analog audio… video audio.… to avoid a conflict.…".

Reply:
thanks for your comment, will fix asap

Reply:
thanks for your comment, will fix asap